integration_tests/integration_dotnet_test.go

```diff
@@ -486,8 +486,8 @@ func TestProvider(t *testing.T) {
 	testDotnetProgram(t, &integration.ProgramTestOptions{
 		Dir:            filepath.Join("provider"),
 		LocalProviders: []integration.LocalDependency{{Package: "testprovider", Path: "testprovider"}},
-		Verbose: true,
-		DebugLogLevel: 10,
+		Verbose:        true,
```

Comment:
Oh these were from debugging, Verbose and DebugLogLevel can be removed from this test.

Reply:
Fixed up in https://github.com/pulumi/pulumi-dotnet/pull/82